src/components/Map/LocateControl.tsx

```diff
@@ -0,0 +1,31 @@
+import * as Locatecontrol from "leaflet.locatecontrol";
+import "leaflet.locatecontrol/dist/L.Control.Locate.css";
+import React, { useEffect } from "react";
+import { useLeaflet } from "react-leaflet";
+
+import "./LocateControl.css";
+
+type Props = {
+  container: React.RefObject<HTMLInputElement>;
+};
+
+const LocateControl = ({ container }: Props) => {
+  const { map } = useLeaflet();
+
+  useEffect(() => {
+    // FIXME the icons are from fontawesome by default, override them by setting the options below
+    const locatecontrol = new Locatecontrol({
+      strings: {
+        title: "Standort"
+      }
+      /*icon: "locate-control-icon", iconLoading: "locate-control-icon-loading"*/
+    });
```

Comment:
Ist der FIXME comment von dir? Müssen wir stattdessen ein Icon aus Figma nutzen?

Reply:
Müssen nicht, können aber. Und ja der ist von mir :D 
Ich hab die CSS File schonmal angelegt und die Klassen hier auskommentiert, ich denke wir werden da eigene Icons benutzen, da müsste nur noch definiert werden wie der Loading Spinner aussehen soll. Wenn du willst, kannst du mir da auch helfen, da ich das mit dem Custom Icon und dem styling noch nicht ganz perfekt hinbekommen habe.

Reply:
Helfe gerne. Wollen wir dafür vllt ein Issue anlegen? Würde mich dann in einem separaten PR darum kümmern und mir das mal ansehen.

Reply:
Gerne!